core/silkworm/execution/processor.cpp

```diff
@@ -113,13 +117,11 @@ Receipt ExecutionProcessor::execute_transaction(const Transaction& txn) noexcept
 
     cumulative_gas_used_ += gas_used;
 
-    return {
-        txn.type,                       // type
-        vm_res.status == EVMC_SUCCESS,  // success
-        cumulative_gas_used_,           // cumulative_gas_used
-        logs_bloom(state_.logs()),      // bloom
-        state_.logs(),                  // logs
-    };
+    receipt.type = txn.type;
+    receipt.success = vm_res.status == EVMC_SUCCESS;
```

Comment:
Any chance `vm_res.status != EVMC_SUCCESS` ?
If so ... can we exit the function early (right after `evm_.execute`) ?

Reply:
Ditto: blocks with unsuccessful transactions are still valid.

Reply:
Ok